### DISCUSSION TITLE
the word OUTER in LEFT OUTER JOIN, RIGHT OUTER JOIN and FULL OUTER JOIN does not change the semantics of the JOIN

### DISCUSSION THREAD

**Discussion by juandent** [fnc12/sqlite_orm#918] [Show and tell]
Except that FULL OUTER JOIN and RIGHT OUTER JOIN are not supported directly by SQLite.

So LEFT OUTER JOIN is equivalent to LEFT JOIN
RIGHT OUTER JOIN to RIGHT JOIN
FULL OUTER JOIN to FULL JOIN

This was taken from page 136 of "A Guide to the SQL Standard" fourth edition by C.J. Date with Hugh Darwen. Addison Wesley 1997.

**Comment by data-man** [comment]
> Except that FULL OUTER JOIN and RIGHT OUTER JOIN are not supported directly by SQLite.

https://github.com/sqlite/sqlite/commit/94e615a6acf3a1519d98e85a6665c712edb5ea8a :tada:

**Comment by juandent** [comment]
But both outer joins can be implemented 

See the guide of sqlite_orm

Juan Dent
